utils/installer/install.ps1

```diff
@@ -0,0 +1,155 @@
+#Set Variable to master is not set differently
+if ($null -eq $LVBRANCH) {
+    $LVBRANCH = "master"
+}
+
+#set -o nounset # error when referencing undefined variable
+$ErrorActionPreference = "Stop" # exit when command fails
+
+function mkdirondemand($path) {
+	if ((Test-Path $path) -eq $false) {
+		mkdir $path
+	}
+}
+
+function commandinstalled($cmd){
+	try { Get-Command $cmd > $null }
+	catch { return $false }
+	return $true
+}
+
+function moveoldlvim {
+	Write-Output "Not installing LunarVim"
+	Write-Output "Please move your $HOME\.local\share\lunarvim folder before installing"
+	exit
+}
+
+function asktoinstallpip {
+	Write-Output "Please install pip3 before continuing with install"
+	exit
+}
+
+function installnode {
+	if ((commandinstalled("winget.exe")) -eq $false) {
+		Write-Output "winget is not installed, but is the only package manager currently supported."
+		Write-Output "node is not installed. Please install it manually"
+		exit
+	}
+	winget.exe install "OpenJS.NodeJSLTS"
+}
+
+function asktoinstallnode {
+	Write-Output "node not found"
+	$answer = Read-Host "Would you like to install node now (y/n)" -Confirm
+	if($answer.ToLower() -eq 'y') {
+		installnode
+	}
+}
+
+function installpynvim {
+	Write-Output "Installing pynvim..."
+	pip3.exe install pynvim --user
+}
+
+function installpacker {
+	git clone https://github.com/wbthomason/packer.nvim ${HOME}/.local/share/lunarvim/site/pack/packer/start/packer.nvim
+}
+
+function modifyhomevar($config) {
+	
+	Copy-Item $config $config".save"
+	(Get-Content -Path $config) |
+		ForEach-Object {$_ -Replace 'os.getenv "HOME"', 'os.getenv "HOMEDRIVE" .. os.getenv "HOMEPATH"'} |
+			Set-Content -Path $config
+}
```

Comment:
I'm not sure I understand what this function is supposed to do.

Reply:
This function replaces every appearance of 'os.getenv "HOME" in the passed lua file.
I did this because in my tests lua could not fetch a environment variable home because on windows it does not exist. only in powershell you can use $HOME. The variables HOMEDRIVE and HOMEPATH however do exist and concatenated make up $HOME